src/inthandler.S

```diff
@@ -358,6 +365,6 @@ save_fpu_regs:
 
 
 	.section .bss
- 	.align 8
+ 	.align 8 # TODO: align to 256 bytes, is that intended?
```

Comment:
No I'd say it's a bug, I guess we could even remove the line but I'm not sure about the semantics of `.lcomm` so maybe let's keep at least a 4-byte alignment

```suggestion
        .align 2
```

Reply:
You could explicitly choose `.balign` (bytes) or `.p2align` (powers of two) to make this more explicit.

Reply:
Edited suggestion to use p2align

Reply:
https://sourceware.org/binutils/docs-2.40/as.html#Align
> GAS also provides .balign and .p2align directives, described later, which have a consistent behavior across all architectures (**but are specific to GAS**). 

Is it fine to only support / rely on GNU as?

Reply:
libdragon only compiles with its own toolchain, which is a modern GCC. So yes.

Reply:
I changed the other `.align`s in inthandler.S to `.p2align`, there are others elsewhere I left alone (just to try to be consistent in separate files)